app/models/iiif_playlist_canvas_presenter.rb

```diff
@@ -13,19 +13,26 @@
 # ---  END LICENSE_HEADER BLOCK  ---
 
 class IiifPlaylistCanvasPresenter
-  attr_reader :playlist_item, :stream_info
+  attr_reader :playlist_item, :stream_info, :cannot_read_item
   attr_accessor :media_fragment
 
-  def initialize(playlist_item:, stream_info:, media_fragment: nil)
+  def initialize(playlist_item:, stream_info:, cannot_read_item: nil, media_fragment: nil)
```

Comment:
How about making the default for the new kwarg be `false` to better show it is expected to be a boolean?
```suggestion
  def initialize(playlist_item:, stream_info:, cannot_read_item: false, media_fragment: nil)
```